config.go

```diff
@@ -411,6 +411,8 @@ type Config struct {
 
 	RejectHTLC bool `long:"rejecthtlc" description:"If true, lnd will not forward any HTLCs that are meant as onward payments. This option will still allow lnd to send HTLCs and receive HTLCs but lnd won't be used as a hop."`
 
+	NoRejectPositiveInboundFees bool `long:"no-reject-positive-inbound-fees" description:"If true, lnd will also accept positive inbound fees. By default, lnd only accepts negative inbound fees (an inbound \"discount\") to keep the channels open for senders that do not recognize inbound fees."`
```

Comment:
:s/to keep the channels open for senders that do not recognize inbound fees./to remain backwards compatible to older clients and implementations which do not support inbound fees yet (CLN, ..)./g

Reply:
included